src/modules/extendedCallList/i18n/nl_NL.root.json

```diff
@@ -262,6 +262,7 @@
             "credits": "Credits",
             "default": "Geen sortering",
             "distance_dispatch": "Distance to next dispatch center",
+            "distance_station": "Distance to next station",
```

Comment:
"distance_dispatch": "Afstand tot dichtsbijzijnde meldkamer",
            "distance_station": "Afstand tot dichtsbijzijnde post",

Reply:
made a typo, changed it.